nextstrain/cli/util.py

```diff
@@ -24,3 +28,33 @@ def colored(color, text):
         end   = colors["reset"],
         text  = text,
     )
+
+
+def check_for_new_version():
+    newer_version = new_version_available()
+
+    if newer_version:
+        print("A new version of nextstrain-cli, %s, is available!  You're running %s." % (newer_version, __version__))
+        print()
+        print("Upgrade by running:")
+        print()
+        print("    pip install --upgrade nextstrain-cli")
+        print()
+
```

Comment:
Can you add an `else` statement to print "the nextstrain CLI is up-to-date" (or similar) here.

Reply:
Updated with your suggestion, as well as [some additional wording at the end of the `update` command](https://github.com/nextstrain/cli/pull/18/commits/818c1eca1a06905169e6ddd4d7622b34452ae8dc#diff-c6baea57f92567a217fe049316ce37a0R34).